src/clarity_vm/clarity.rs

```diff
@@ -743,6 +758,14 @@ impl<'a, 'b> ClarityTransactionConnection<'a, 'b> {
         identifier: &QualifiedContractIdentifier,
         contract_content: &str,
     ) -> Result<(ContractAST, ContractAnalysis), Error> {
+        let epoch_id = self
+            .with_clarity_db_readonly(|db| db.get_current_stacks_epoch())
+            .expect("Failed to load current epoch")
+            .epoch_id;
+
+        // ClarityVersionPragmaTodo: need to use contract's declared version or default
```

Comment:
Is there an issue somewhere tracking these TODOs, if they're not going to happen in this PR?

Reply:
Opened https://github.com/blockstack/stacks-blockchain/issues/2719